image-layout.md

```diff
@@ -162,12 +162,12 @@ Those tags will often be represented in an image-layout repository with matching
 
 ### Index Example
 
-```json,title=Manifest%20List&mediatype=application/vnd.oci.image.manifest.list.v1%2Bjson
+```json,title=Manifest%20List&mediatype=application/vnd.oci.image.index.v1%2Bjson
 {
   "schemaVersion": 2,
   "manifests": [
     {
-      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "mediaType": "application/vnd.oci.image.index.v1+json",
       "size": 7143,
       "digest": "sha256:0228f90e926ba6b96e4f39cf294b2586d38fbb5a1e385c05cd1ee40ea54fe7fd",
       "annotations": {
```

Comment:
yes. Just like a file `./refs/stable` could be a descriptor of a manifest-list, this named reference points to a now image-index (previously manifest-list). I only changed this example to demo an image-index, rather than only a manifest.